catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -21,6 +21,14 @@
             name="Directory Path" id="monitoredDirectoryPath" required="true"
             type="String" default=""/>
 
+        <AD description="Specifies the number of threads to use when monitoring"
+            name="Thread Pool Size" id="numThreads" required="true"
+            type="Integer" default="1"/>
+
+        <AD description="Specifies the time to wait (in milliseconds) before acquiring a lock on a file in the monitored directory.  This interval is used for sleeping between attempts to acquire the read lock on a file to be ingested.  The default value of 1 second is recommended."
```

Comment:
This is a great description btw. Although I would say a "default value of 1000 miliseconds" to keep the units consistent. Also fix the default value to match.